Xamarin.Forms.Platform.UAP/Resources.xaml

```diff
@@ -5,6 +5,7 @@
 	x:Class="Xamarin.Forms.Platform.UWP.Resources">
 
 	<ResourceDictionary.MergedDictionaries>
+        <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
```

Comment:
Consider doing this programmatically. That way you can throw if WinUI wasn't referenced by the head project (which could be an issue in VS2017). It would even allow you to only throw for those controls that rely on WinUI, thus making it non-breaking and only required for RefreshView.

Reply:
I think it's a good idea, what do you think @PureWeen?

Reply:
`MergedDictionaries.Add(new Microsoft.UI.Xaml.Controls.XamlControlsResources());`

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/7319/files#diff-fadef492a65644f17c0a99fa2e742e80R39